tests/foreman/cli/test_setting.py

```diff
@@ -226,8 +226,17 @@ def test_positive_update_email_delivery_method_sendmail():
 
     :CaseImportance: Low
 
-    :CaseAutomation: NotAutomated
+    :CaseAutomation: Automated
     """
+    sendmail_argument_value = gen_string('alphanumeric')
+    arguments = {
```

Comment:
`email_config_params` or `sendmail_config_params` would be much appropriate than `arguments`.

Reply:
Done